slatedb/src/mem_table_flush.rs

```diff
@@ -179,6 +179,7 @@ impl MemtableFlusher {
                 })?;
             }
             imm_memtable.notify_flush_to_l0(Ok(()));
+            self.db_inner.db_stats.immutable_memtable_flushes.inc();
```

Comment:
should we move it after the manifest write? the flush isn't successful until the l0 is in the manifest.

Reply:
actually nvm from the pov of the writer its flushed, though on a restart it may need to be reflushed. This seems ok to me